common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -52,7 +52,9 @@ import (
 // R represents row type in executions table, valid values are:
 // R = {Shard = 1, Execution = 2, Transfer = 3, Timer = 4, Replication = 5}
 const (
-	cassandraProtoVersion = 4
+	// ProtoVersion is the protocol version used to communicate with Cassandra cluster
+	ProtoVersion = 4
```

Comment:
This name is really confusing. May be just remove this const completely and hardcode `4` where it is used?

Reply:
how about ProtocolVersion?